.github/workflows/publish-prod.yaml

```
@@ -24,6 +24,8 @@ jobs:
         run: npm run build
         env:
           NODE_OPTIONS: --max_old_space_size=8192
+          DOCS_SITE_URL: https://stage.unsupported.docs.camunda.io
```

oops I screwed this up.

```suggestion
          DOCS_SITE_URL: https://unsupported.docs.camunda.io
```